### ISSUE TITLE
View dictionary not working 

### ISSUE THREAD

**Issue by GodswillOnuoha** [OpenConceptLab/ocl_issues#91]
<img width="1276" alt="screenshot 2019-02-15 at 1 46 27 pm" src="https://user-images.githubusercontent.com/40201026/52859201-62f42400-312c-11e9-8c65-6320040df903.png">

clicking Mitama, Kampala Hospital or Mulago hospital gives the following screen

<img width="804" alt="screenshot 2019-02-15 at 1 43 06 pm" src="https://user-images.githubusercontent.com/40201026/52859409-eada2e00-312c-11e9-9af9-b5fcefabcfd2.png">

**Comment by GodswillOnuoha** [comment]
There is a button on the OCL-client (view released dictionary) which redirects to the link above. was working initially, but have been showing that error screen for a while

**Comment by rkorytkowski** [comment]
Please include hyperlinks when reporting errors and referring to any pages so we don't have to write them down from screenshots or click through UI. It's just a few seconds extra for you to copy & paste.

**Comment by rkorytkowski** [comment]
https://openmrs.qa.openconceptlab.org/dictionaryOverview/users/admin/collections/d3h7d3e4g6d3e4/ 

https://qa.openconceptlab.org/users/admin/collections/d3h7d3e4g6d3e4/

**Comment by rkorytkowski** [comment]
Sentry: https://sentry.io/organizations/openconceptlab/issues/739148593/?project=246148&query=is%3Aunresolved&statsPeriod=14d&utc=true

Server logs: 
```
>> ERROR 2019/02/21 01:31:07 [base base.py:226 handle_uncaught_exception()] Internal Server Error: /users/admin/collections/d3h7d3e4g6d3e4/
web_1     | >> Traceback (most recent call last):
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 139, in get_response
web_1     | >>     response = response.render()
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/response.py", line 105, in render
web_1     | >>     self.content = self.rendered_content
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/response.py", line 82, in rendered_content
web_1     | >>     content = template.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 140, in render
web_1     | >>     return self._render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/test/utils.py", line 85, in instrumented_test_render
web_1     | >>     return self.nodelist.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 840, in render
web_1     | >>     bit = self.render_node(node, context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 854, in render_node
web_1     | >>     return node.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
web_1     | >>     return compiled_parent._render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/test/utils.py", line 85, in instrumented_test_render
web_1     | >>     return self.nodelist.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 840, in render
web_1     | >>     bit = self.render_node(node, context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 854, in render_node
web_1     | >>     return node.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 123, in render
web_1     | >>     return compiled_parent._render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/test/utils.py", line 85, in instrumented_test_render
web_1     | >>     return self.nodelist.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 840, in render
web_1     | >>     bit = self.render_node(node, context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 854, in render_node
web_1     | >>     return node.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
web_1     | >>     result = block.nodelist.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 840, in render
web_1     | >>     bit = self.render_node(node, context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 854, in render_node
web_1     | >>     return node.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 62, in render
web_1     | >>     result = block.nodelist.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 840, in render
web_1     | >>     bit = self.render_node(node, context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 854, in render_node
web_1     | >>     return node.render(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 1195, in render
web_1     | >>     resolved_args, resolved_kwargs = self.get_resolved_arguments(context)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 1041, in get_resolved_arguments
web_1     | >>     resolved_args = [var.resolve(context) for var in self.args]
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/base.py", line 611, in resolve
web_1     | >>     new_obj = func(obj, autoescape=context.autoescape, *arg_vals)
web_1     | >>   File "/usr/local/lib/python2.7/site-packages/django/template/defaultfilters.py", line 509, in join
web_1     | >>     value = map(force_text, value)
web_1     | >> TypeError: argument 2 to map() must support iteration


```

**Comment by rkorytkowski** [comment]
It appears that the dictionary has been created without required supported_locales value. It can be fixed through UI by going to https://qa.openconceptlab.org/users/admin/collections/d3h7d3e4g6d3e4/edit/, setting Supported Locales and updating.

#96 needs to be addressed to prevent creation of such malformed sources/collections.

**Comment by rkorytkowski** [comment]
Before we decide how to fix #96, please create sources/collections with supported_locales: ['en'] or whatever makes sense.
